lib/stripe/account.rb

```diff
@@ -79,7 +79,7 @@ def serialize_params_account(obj, update_hash)
       update_hash
     end
 
-    def protected_fields
+    def self.protected_fields
```

Comment:
Moved to class level so that we can access the field from other class-level methods (like update).